src/components/elements/FilterChips.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import PropTypes from 'prop-types'
+import Chip from 'material-ui/Chip'
+import { removeFilter  } from '../../actions/filterActions'
+
+const FilterChips = ({dispatch, filter, filterType, filterOptions, lookupMethod, history}) => {
+    if (readyToRender(filter, filterOptions)) {
+        let selectedFilters = extractWantedFilter(filterType, filter)
+        return (
+            <div style={chipStyles.wrapper}>
+                {selectedFilters.map((selectedFilter) => renderChip(dispatch, history, filter, selectedFilter, filterOptions, lookupMethod))
+                }
+            </div>
+        )
+    }
+    else {
+        return <div></div>
+    }
+
+}
+
+const renderChip = (dispatch, history, filter, filterOptions, selectedFilterType, lookupMethod) => {
+    let filterType= lookupMethod(selectedFilterType)(filterOptions)
+    return (
+            <Chip
+                key={filterType.id}
+                onRequestDelete={() => dispatch(removeFilter(history)(filter, filterType))}
+                style={chipStyles.chip}
+            >
+                {filterType.name}
+            </Chip>
+        )
+}
+
+const readyToRender = (filter, filterTypes) => {
+    return filter && filter.eventTypes && filterTypes
+}
+
+const extractWantedFilter = (selectedFilterType, filter) => {
+    return selectedFilterType === 'eventType' ? filter.eventTypes : filter.eventTypes
+}
+
+
+const chipStyles = {
+    chip: {
+        margin: 4
+    },
+    wrapper: {
+        display: 'flex',
+        flexWrap: 'wrap'
+    }
+}
+
+FilterChips.propTypes = {
+    dispatch: PropTypes.func,
+    history: PropTypes.object,
+    filter: PropTypes.object,
+    filterType: PropTypes.string,
+    filterOptions: PropTypes.object,
+    lookupMethod: PropTypes.func,
+}
+
+export const mapStateToProps = (state, ownProps) => {
+    return {
+        ...ownProps,
+        filter: state.events.filter ? state.events.filter : {},
```

Comment:
Reducers should always exist; if they don't, there's a major problem and we should want an error to occur so that we're not silently failing to find a path that's been moved or mistyped, in my opinion.